hack/create-cluster.sh

```diff
@@ -14,27 +14,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -euo pipefail
+set -o errexit -o nounset -o pipefail
 
-: "${VM_DRIVER:=virtualbox}"
+: "${VM_DRIVER:=docker}"
+: "${VM_CPUS:=2}"
 : "${VM_MEMORY:=$(( 1024 * 4 ))}"
+: "${KUBERNETES_VERSION:=v1.15.12}"
 
 if [[ "$(minikube status)" != *"Running"* ]]; then
-    set -x
+    set -o xtrace
     minikube start \
       --vm-driver="${VM_DRIVER}" \
+      --cpus="${VM_CPUS}" \
       --memory="${VM_MEMORY}" \
-      --kubernetes-version=v1.11.3 \
-      --bootstrapper=kubeadm
+      --kubernetes-version="${KUBERNETES_VERSION}"
 else
-    echo "Using current running instance of Minikube..."
-    set -x
+    >&2 echo "A Minikube instance is already running..."
+    exit 1
 fi
 
-minikube addons enable heapster
+catalog_repository="svc-cat"
+catalog_release="catalog"
+catalog_namespace="svc-cat"
+helm repo add "${catalog_repository}" https://svc-catalog-charts.storage.googleapis.com
+kubectl create namespace "${catalog_namespace}"
+helm install "${catalog_release}" \
+  --namespace "${catalog_namespace}" \
+  "${catalog_repository}/${catalog_release}"
 
-kubectl apply -f https://raw.githubusercontent.com/Azure/helm-charts/master/docs/prerequisities/helm-rbac-config.yaml
-helm init --service-account tiller --wait
-
-helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
-helm upgrade --install catalog --namespace svc-cat svc-cat/catalog --wait
+set +o xtrace
+while kubectl get pods --namespace svc-cat --selector "release=${catalog_release}" 2>&1 \
+        | grep "No resources found in ${catalog_namespace} namespace."; do
```

Comment:
I'm not _quite_ happy with grepping for error strings: they might start localizing the CLI some day.  Perhaps something like:
```suggestion
while [[ "$(kubectl get pods --namespace "${catalog_namespace}" --selector "release=${catalog_release}" --output=go-template='{{.items||len}}')" == 0 ]]; do
```
… But then, it might be harder to read…
(Also, you missed a `svc-cat`)